test/test_common/network_utility.h

```diff
@@ -38,19 +38,22 @@ Address::InstanceConstSharedPtr findOrCheckFreePort(const std::string& addr_port
                                                     Address::SocketType type);
 
 /**
- * Get a URL ready IP loopback address as a string.
+ * Get a URL ready IP loopback address as a string. Notes ipv4 loopback address is not unique.
  * @param version IP address version of loopback address.
+ * @param host_id additional parameter to generate series of loopback addresses for ipv4.
```

Comment:
Unclear from this param name and text what it does. Can you clarify? Same below.